tests/lib/snaps/snapctl-disable-services-install-hook/meta/hooks/install

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+for service in simple forking do;
+    echo "Disabling $service service from install hook"
+    if [ "$(snapctl stop --disable snapctl-disable-services-install-hook.$service 2>&1)" != "" ]; then
```

Comment:
It's already running with `set -e`, so I believe the whole script could be this:
```
#!/bin/bash -e

for service in simple forking; do
    echo "Disabling $service service from install hook"
    snapctl stop --disable "snapctl-disable-services-install-hook.$service"
done
```

Reply:
Fixed